### PR TITLE
Visual Reader

### PR DESCRIPTION
**Pull request by adityashah1212**
https://github.com/evuz/VisualComicReader

### CODE REVIEW

**Comment by TingPing** [comment]
bot, build org.evuz.visualReader

File: org.evuz.visualReader.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>org.evuz.visualReader.desktop</id>
+    <name>Visual Comic Reader</name>
+    <developer_name>evuz</developer_name>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>ISC</project_license>
+    <summary>Read comics in differents platforms but the same way</summary>\
+</component>
```
Comment (TingPing) [inline org.evuz.visualReader.appdata.xml]: Could use more content like screenshots, release info, etc. And ideally see if upstream would be interested in it.

**Comment by TingPing** [comment]
bot, build org.evuz.visualReader

File: org.evuz.visualReader.json [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                "mkdir -p /app/main /app/bin",
+                "cp -ra main/* /app/main/",
+                "install -Dm755 VisualComicReader /app/bin/",
+                "install -Dm644 icon-256x256.png /app/share/icons/hicolor/256x256/apps/org.evuz.visualReader.png",
+                "install -Dm644 icon-128x128.png /app/share/icons/hicolor/128x128/apps/org.evuz.visualReader.png",
+                "install -Dm644 icon-64x64.png /app/share/icons/hicolor/64x64/apps/org.evuz.visualReader.png",
+                "install -Dm644 icon-48x48.png /app/share/icons/hicolor/48x48/apps/org.evuz.visualReader.png",
+                "install -Dm644 icon-32x32.png /app/share/icons/hicolor/32x32/apps/org.evuz.visualReader.png",
+                "install -Dm644 icon-16x16.png /app/share/icons/hicolor/16x16/apps/org.evuz.visualReader.png",
+                "install -Dm644 org.evuz.visualReader.desktop /app/share/applications",
```
Comment (nedrichards) [inline org.evuz.visualReader.json]: This needs to be `install -Dm644 org.evuz.visualReader.desktop /app/share/applications/org.evuz.visualReader.desktop `

Comment (nedrichards) [reply org.evuz.visualReader.json]: @adityashah1212 do you know if you'll have time to get that fix in?

**Comment by adityashah1212** [comment]
You can close this pull request. I may not be able to maintain it.

**Pull request closed without merging**